internal/backends/winit/lib.rs

```diff
@@ -389,6 +397,19 @@ impl WinitWindowAccessor for i_slint_core::api::Window {
     ) -> Option<T> {
         winit_window_rc_for_window(self).as_ref().map(|w| callback(w))
     }
+
+    fn on_winit_window_event(
+        &self,
+        mut callback: impl FnMut(&winit::event::WindowEvent) -> bool + Send + 'static
+    ) {
+        i_slint_core::window::WindowInner::from_pub(&self)
+            .window_adapter()
+            .internal(i_slint_core::InternalToken)
+            .and_then(|wa| wa.as_any().downcast_ref::<WinitWindowAdapter>())
+            .map(|adapter| {
```

Comment:
I didn't have this return an error since `WinitWindowAccessor::with_winit_window` doesn't either, and I'm not sure what error type would be best. I could certainly add a note to the rustdoc about it being a no-op for non-winit windows for now.

Reply:
Maybe a "full" signature would be `Result<Option<Box<dyn FnMut(...) -> ...>>, slint::PlatformError>` to allow the caller to implement chaining?